README.md

```diff
@@ -58,16 +58,23 @@ The following command can be executed with powershell:
     docker compose up -d
 ```
 
-Wanneer er wijzigingen zijn in de projecten en er opnieuw een schoon build (image) moet worden aangemaakt, dan kan de volgende commando worden uitgevoerd:
+When there are changes to the project, then a new build (image) must be created. This can be done with the command: 
 
 ```sh
     docker-compose up --build --force-recreate -d 
 ```
+
 of
```

Comment:
deze 'of' mag ook nog naar het Engels ('or') worden vertaald ;)

Reply:
Done!